JobToCandidateRecs.md

```diff
@@ -0,0 +1,73 @@
+Job To Candidate Recommendation Service
+=========================================
+
+- [Summary](#summary)
+- [Request Response Structure](#request-response-structure)
+- [Table Structure](#table-structure)
+
+## Summary
+DSAD's job-to-candidate-recs service is a FastAPI based web service, which takes a `job_id` and its related `latitude` 
+and `longitude`, and will return a list of `Candidate` within 50 miles of `job_id` location. 
+
+The related dynamodb tables are `JobCandidateGeo` and `CaroteneCandidateGeo`. 
```

Comment:
Don't need to mention these tables since its an implementation detail that isn't important to the user.